lib/handlerTemplate.js

```diff
@@ -192,7 +192,11 @@ function prepareRequestChain(conf) {
         if (requestConf.request) {
             template = new Template(requestConf.request);
             requestSpec.request = function(restbase, context) {
-                var req = template.eval(context);
+                var req = template.expand(context);
+                if (!requestConf.request.method) {
+                    // Do we really want this?
+                    req.method = context.request.method || 'get';
```

Comment:
Actually, since in templating code we are already making a fallback (`spec.method = "'" + (spec.method || 'get') + "'";`), this branch would never execute. Or am I missing something?